src/docker/compose.dss-local.yaml

```diff
@@ -24,6 +26,7 @@ services:
 
   postgrest:
     depends_on:
+      - diesel
       - postgres
     environment:
       PGRST_DB_URI: "postgres://econia:econia@postgres:5432/econia"
```

Comment:
you'll have to update the role down here to be `web_anon`
The reason this worked is that `anon` is a role that I created that can see anything (see the first migration)